dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java

```diff
@@ -249,6 +261,22 @@ public TaskResponse run(String execCommand, TaskCallBack taskCallBack) throws Ex
             }
         }
 
+        if (podLogOutputFuture != null) {
+            try {
+                // Wait kubernetes pod log collection finished
+                String msg = (String) podLogOutputFuture.get();
+                if (StringUtils.isEmpty(msg)) {
+                    // delete pod after successful execution
+                    ProcessUtils.cancelApplication(taskRequest);
+                } else {
+                    // contains error message
+                    logger.error(msg);
```

Comment:
just error msg?

Reply:
`msg` here can only be `The driver pod does not exist.`